.devcontainer/devcontainer.json

```diff
@@ -3,24 +3,20 @@
     "name": "ros2_rust_roscon_spain:latest",
     "build": {
     "context": "..",
-    "dockerfile": "../.docker/Dockerfile",
+    "dockerfile": "../docker/Dockerfile",
     "args": {
       "XDG_RUNTIME_DIR": "${localEnv:XDG_RUNTIME_DIR}"
     }
     },
-    "workspaceFolder": "/ros2_rust_ws",
```

Comment:
devcontainers automatically use /workspaces/YOUR_PROJECT in the container

Even though using the workspaceFolder variable makes it explicit, you also diverge from the default behavior. I prefer to keep it as is.

Reply:
Makes sense and this is a good point, I just wanted to use the same workspace for the overall workshop, to avoid using a different one for devcontainers, docker or host. But maybe I can find a way to explain everything using a common workspaceFolder name. Thank you!